app/components/github-label.js

```diff
@@ -1,26 +1,13 @@
 import Component from '@ember/component';
 import { computed } from '@ember/object';
 import { htmlSafe } from '@ember/string';
-import githubEmojis from 'ember-help-wanted/constants/github-emojis';
 import invert from 'invert-color';
 
 export default Component.extend({
   classNames: ['github-label'],
   tagName: 'span',
   attributeBindings: ['style'],
 
-  labelWithEmoji: computed('label', function() {
```

Comment:
I moved this into a helper so we can use it on a different places and also test it easier.